### PR TITLE
Used the modified camelCase name convention

### PR DESCRIPTION
**Pull request by Rohit25negi**
making the name convention consistent. Raised [here](https://github.com/dimagi/commcare-hq/commit/2f750ae609bc65fe76750056cd4c0b87098490d7) 

@mkangia @orangejenny

### CODE REVIEW

File: corehq/apps/data_interfaces/forms.py [hunk 1]
```diff
@@ -86,11 +86,11 @@ def validate_case_property_name(value, allow_parent_case_references=True):
     return value
 
 
-def hidden_bound_field(field_name):
+def hidden_bound_field(field_name, data_value):
     return Field(
         field_name,
         type='hidden',
-        data_bind='value: %s' % field_name,
+        data_bind='value: %s' % data_value,
```
Comment (mkangia) [inline corehq/apps/data_interfaces/forms.py]: nit: can consider renaming it to `field_value_attr` or something better
@orangejenny might have better suggestion

**Review by mkangia — APPROVED** [review]

**Comment by Rohit25negi** [comment]
merging it as its urgent

**Comment by Rohit25negi** [comment]
@mkangia @orangejenny  do we have test coverage for this one?

**Comment by Rohit25negi** [comment]
Merging this. test cases passed locally for `corehq.apps.data_interfaces.tests`.

**Pull request merged**